src/client.rs

```diff
@@ -66,9 +66,9 @@ impl Client {
         gtfs_filename: &str,
         producer_id: &str,
         producer_name: &str,
-    ) -> Result<(), failure::Error> {
-        let gtfs = gtfs_structures::RawGtfs::from_zip(gtfs_filename)?;
-        let routes = gtfs.routes?;
+    ) -> Result<(), anyhow::Error> {
+        let gtfs = gtfs_structures::RawGtfs::from_zip(gtfs_filename).map_err(|e| e.compat())?;
```

Comment:
So on the longer term, it would be nice to also remove failure from gtfs_structures?

Reply:
yes, but it's maybe a bit early to do this, there are many contenders for the reference error management crate :stuck_out_tongue_winking_eye: